src/tasks/AppleAppBuilder/Templates/runtime.m

```diff
@@ -261,8 +253,34 @@ static int32_t load_icu_data ()
     const char* bundle = get_bundle_path ();
     chdir (bundle);
 
+    char icu_dat_path[1024];
+    int res;
+
+    res = snprintf (icu_dat_path, sizeof (icu_dat_path) - 1, "%s/%s", bundle, "icudt.dat");
+    assert (res > 0);
+
     // TODO: set TRUSTED_PLATFORM_ASSEMBLIES, APP_PATHS and NATIVE_DLL_SEARCH_DIRECTORIES
-    monovm_initialize(0, NULL, NULL);
+#if !INVARIANT_GLOBALIZATION
```

Comment:
Add TODO: to move this to runtimeconfig.json once it's supported

Reply:
This doesn't need to be in runtimeconfig.json - or at least, it shouldn't be in the parts of `runtimeconfig.json` that are created by users.  It is part of the host's responsibility to setup the environment for the runtime. It belongs in the embedding code.